### PR TITLE
Ml/usertype registration

### LINKED ISSUE #21: Add switch for Donor/Client on user registration
Need to add a radio button for Donor or Client when registering a new user.

Upon successful registration, the user should be taken to an appropriate landing page (Donation Management or Assistance Request).

### PR DESCRIPTION
**Pull request by maxlemieux**
Added a switch on the user registration page to select 'donor' or 'client'. Landing page then varies based on which type of user is logging in.

This branch ended up also adding seed files. Run the seeds with: 

`sequelize db:seed:all`

### CODE REVIEW

File: routes/html-routes.js [hunk 1]
```diff
@@ -17,6 +21,11 @@ module.exports = (app) => {
     // If the user already has an account send them to the members page
     if (req.user) {
       res.redirect('/members');
+      // if (req.user.user_type === 'client') {
```
Comment (cwalker226) [inline routes/html-routes.js]: are we going to need the commented code later?

File: server.js [hunk 2]
```diff
@@ -31,8 +31,22 @@ require('./routes/api/donate-routes.js')(app);
...
-db.sequelize.sync().then(() => {
-  app.listen(PORT, () => {
-    console.log('==> 🌎  Listening on port %s. Visit http://localhost:%s/ in your browser.', PORT, PORT);
+// db.sequelize.sync().then(() => {
+//   app.listen(PORT, () => {
+//     console.log('==> 🌎  Listening on port %s. Visit http://localhost:%s/ in your browser.', PORT, PORT);
+//   });
+// });
+
+db.User.sync().then(() => {
```
Comment (cwalker226) [inline server.js]: curious why we needed to nest all these?

**Review by cwalker226 — APPROVED** [review]

File: routes/html-routes.js (hunk 1 above)
Comment (maxlemieux) [reply routes/html-routes.js]: no, i've made a new commit to remove this section

File: server.js (hunk 2 above)
Comment (maxlemieux) [reply server.js]: we do not - it was an attempt to get foreign keys working - thanks

**Pull request merged**